src/utils/themes/light_theme.ts

```diff
@@ -176,6 +176,11 @@ export const LIGHT_THEME: Theme = {
       visible: true,
     },
     line: {
+      stroke: '#98A2B3',
+      strokeWidth: 1,
+      visible: true,
+    },
+    crossLine: {
       stroke: '#777',
```

Comment:
The `line.stroke` and `crossLine.stroke` could use the same color. I think the `#98A2B3` has more contrast. 

```suggestion
      stroke: '#98A2B3',
```

Reply:
Just to note: these styles are only default to elastic charts. If you want this change reflected in kibana you'd need to update the eui charts theme.

https://github.com/elastic/eui/blob/d16d296ae4d06a6337a4a2534f4d3bfacff932ea/src/themes/charts/themes.ts#L172-L181

Reply:
done in #5e8f506